config/manifests/bases/splunk-operator.clusterserviceversion.yaml

```diff
@@ -274,5 +274,5 @@ spec:
   provider:
     name: Splunk Inc.
     url: www.splunk.com
-  replaces: splunk-operator.v2.1.0
-  version: 2.2.0
+  replaces: splunk-operator.v2.2.0
```

Comment:
Also this one

Reply:
we are good we do not use it directly , we need to create separate one for openshift and operatorhub.